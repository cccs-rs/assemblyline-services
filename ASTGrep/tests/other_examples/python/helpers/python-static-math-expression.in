(87682403 + 162430398)
731709771 ^ 37620823 ^ ~-698551615
~-(94215832 ^ 94215839)
85882548 ^ 522703947 ^ 773889760 + -333904896
~-487568492 ^ (961067106 ^ call(608635943))
34.6
~-208064968 ^ (var ^ 411025184)
226890904 ^ 739709111 ^ 340051771 + 223100120
11
332 > 77
-12
-(45+6)
not 44 # TODO: handle this safely
