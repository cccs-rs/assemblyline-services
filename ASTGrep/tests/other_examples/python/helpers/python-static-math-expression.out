250112801
34
6
31
~-487568492 ^ (961067106 ^ call(608635943))
34.6
~-208064968 ^ (var ^ 411025184)
60
11
True
-12
-51
not 44
